public/API/API_GetGame.php

```diff
@@ -22,29 +22,31 @@
  *  string     Released                   release date information for the game
  */
 
+use App\Platform\Models\Game;
+
 $gameID = (int) request()->query('i');
```

Comment:
nit: Might be a good time to change this to `$gameId` like the other endpoint.

Reply:
This feedback was erroneously marked as resolved.

Note: I do like marking the feedback I leave as resolved when I go in for a re-review. It helps me quickly take inventory of what I need to check when following up on a review.